pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -319,6 +327,7 @@ def newpkgcb(pkgId, name, arch):
 
             repomd_path = result.path
             repomd = cr.Repomd(repomd_path)
+            repomd_revision = "".join(re.findall('\d', repomd.revision))
```

Comment:
Not the best solution,
For CentOS 8 AppStream, the revision is not a timestamp, it is: `8.1.1911`
http://mirror.linux.duke.edu/pub/centos/8/AppStream/x86_64/kickstart/repodata/repomd.xml
From a quick search about revisions, I saw that sometimes it could be empty:
https://pulp.plan.io/projects/pulp_rpm/repository/9/revisions/54c8837a8e5bd80b9df16689b0cec1c3410e8a61
https://bugzilla.redhat.com/show_bug.cgi?id=1332922